tests/Doctrine/Tests/Models/Cache/Attraction.php

```diff
@@ -18,43 +18,27 @@
 use Doctrine\ORM\Mapping\OneToMany;
 use Doctrine\ORM\Mapping\Table;
 
-/**
- * @Cache("NONSTRICT_READ_WRITE")
- * @Entity
- * @Table("cache_attraction")
- * @InheritanceType("SINGLE_TABLE")
- * @DiscriminatorMap({
- *  1  = "Restaurant",
- *  2  = "Beach",
- *  3  = "Bar"
- * })
- */
+#[Table('cache_attraction')]
+#[Cache('NONSTRICT_READ_WRITE')]
+#[Entity]
+#[InheritanceType('SINGLE_TABLE')]
+#[DiscriminatorMap([1 => 'Restaurant', 2 => 'Beach', 3 => 'Bar'])]
 abstract class Attraction
 {
-    /**
-     * @var int
-     * @Id
-     * @GeneratedValue
-     * @Column(type="integer")
-     */
+    /** @var int */
+    #[Id]
+    #[GeneratedValue]
+    #[Column(type: 'integer')]
     protected $id;
 
-    /**
-     * @psalm-var Collection<int, AttractionInfo>
-     * @Cache
-     * @OneToMany(targetEntity="AttractionInfo", mappedBy="attraction")
-     */
+    /** @psalm-var Collection<int, AttractionInfo> */
+    #[Cache]
+    #[OneToMany(targetEntity: 'AttractionInfo', mappedBy: 'attraction')]
     protected $infos;
 
     public function __construct(
-        /** @Column(unique=true) */
-        protected string $name,
-        /**
-         * @Cache
-         * @ManyToOne(targetEntity="City", inversedBy="attractions")
-         * @JoinColumn(name="city_id", referencedColumnName="id")
-         */
-        protected City $city,
+        #[Column(unique: true)] protected string $name,
+        #[Cache] #[ManyToOne(targetEntity: 'City', inversedBy: 'attractions')] #[JoinColumn(name: 'city_id', referencedColumnName: 'id')] protected City $city,
```

Comment:
My attempt at making this more readable. Are there already PHPCS rules for that?

```suggestion
        #[Column(unique: true)]
        protected string $name,

        #[Cache]
        #[ManyToOne(targetEntity: 'City', inversedBy: 'attractions')]
        #[JoinColumn(name: 'city_id', referencedColumnName: 'id')]
        protected City $city,
```

Reply:
Ugh yeah indeed the output of Rector is not very easy on the eye here :sweat: I'll try to see what phpcs rules currently exist if any.

Reply:
I did some digging and didnt find anything. Opened a feature request here - https://github.com/slevomat/coding-standard/issues/1442 . Will see if we can find some resources in our company to work on this if it turns out that this is not available in some other standard already. No promises tho ;]

Reply:
Awesome! Thanks!

Reply:
@greg0ire @derrabus https://github.com/slevomat/coding-standard/releases/tag/8.6.0 have a look

Reply:
Next step should be to include those in https://github.com/doctrine/coding-standard/blob/10.0.x/lib/Doctrine/ruleset.xml I guess 👍

Reply:
For now, let's stick with the ugly version, and then when we use doctrine/coding-standard 11, it will be pretty again?